### PR TITLE
Ru language 

### PR DESCRIPTION
**Pull request by Iambabyninja**
Add Russian language in web panel

### CODE REVIEW

**Review by celgus — APPROVED** [review]

**Review by fL1pSt3r — APPROVED** [review]
Everything is translated well. But I would translate userDialog.expiryDate as "Дата окончания", but that's a small thing.

**Review by kaixRaf — APPROVED** [review]

**Comment by SaintShit** [comment]
I've merged this to `dev` branch currently

**Comment by master88wi** [comment]
Add Russian language in web panel

**Review by aurora1369 — APPROVED** [review]

**Comment by SokolovOne** [comment]
Good!

**Review by SokolovOne — COMMENTED** [review]
Спасибо огромное!

File: app/dashboard/public/locales/ru.json [hunk 1]
```diff
@@ -0,0 +1,159 @@
...
+
+"revokeUserSub.prompt": "Вы уверены, что хотите отозвать подписку для пользователя <b>{{username}}</b>?",
+"revokeUserSub.title": "Отозвать подписку пользователя",
+"revokeUserSub.success": "Подписка пользователя {{username}} успешно отозвана.",
+"revokeUserSub.error": "Отзыв подписки не удался, пожалуйста, попробуйте еще раз.",
+
+"resetAllUsage.title": "Сбросить расход трафика для всех пользователей",
+
+"resetAllUsage.prompt": "Это действие полностью очищает весь расход трафика пользователей. Вы уверены? ЭТО ДЕЙСТВИЕ НЕОБРАТИМО!",
+"resetAllUsage.success": "расход трафика успешно сброшен.",
```
Comment (SokolovOne) [inline app/dashboard/public/locales/ru.json:144]: "Р"асход

**Review by SokolovOne — COMMENTED** [review]
Очепятка

**Review by SokolovOne — APPROVED** [review]

**Review by SokolovOne — APPROVED** [review]

**Review by marat2509 — CHANGES_REQUESTED** [review]
`"resetAllUsage.success": "расход трафика успешно сброшен.",`
Вместо "расход" должно быть "Расход":
`"resetAllUsage.success": "Расход трафика успешно сброшен.",`

**Review by sblvkr — APPROVED** [review]

**Pull request merged**